staging/src/k8s.io/code-generator/cmd/validation-gen/targets.go

```diff
@@ -220,11 +220,6 @@ func GetTargets(context *generator.Context, args *Args) []generator.Target {
 			return cmp.Compare(a.Name.String(), b.Name.String())
 		})
 
-		// Deterministic ordering helps in logs and debugging.
-		slices.SortFunc(rootTypes, func(a, b *types.Type) int {
```

Comment:
Huh.  I could swear it was coming through in a different order each time, but now it seems to be reliable.

Reply:
I think you did fix a sort problem, it's just that somehow the sort code got duplicated so it was sorting twice in a row (see above lines).

Reply:
OHH!  I must have messed up a rebase or something